app/jobs/close_patients_job.rb

```diff
@@ -30,8 +30,14 @@ def perform(*_args)
         patient[:monitoring_reason] = 'Completed Monitoring (system)'
       end
 
-      # Send closed email to patient if they are a reporter
-      PatientMailer.closed_email(patient).deliver_later if patient.save! && patient.email.present? && patient.self_reporter_or_proxy?
+      # Send closed email or SMS to patient if they are a reporter
+      if patient.save! && patient.self_reporter_or_proxy?
+        if ['SMS Texted Weblink', 'SMS Text-message'].include? patient.preferred_contact_method
+          PatientMailer.closed_sms(patient)
+        elsif patient.email.present?
+          PatientMailer.closed_email(patient).deliver_later(wait_until: patient.time_to_contact_next)
+        end
```

Comment:
Do you have a `else` case in mind? Does the system need to inform that a close message couldn't be sent because the patient had no usable contact method?

Reply:
As in should it make a note in the summary email about the method of the closed message/say if it couldn't send because of a blank email address?

Reply:
Any other time the system fails to contact a patient there is a History item. I'm talking about preserving that pattern.

Reply:
This should be addressed now